bower.json

```diff
@@ -24,6 +24,7 @@
     "purescript-partial": "master",
     "purescript-prelude": "master",
     "purescript-st": "master",
+    "purescript-safe-coerce": "master",
```

Comment:
This pull request doesn’t use Safe.Coerce.coerce anymore, we can remove this dependency.

Reply:
Fixed.